### PR TITLE
Od/redis

### PR DESCRIPTION
**Pull request by orlando**
This PR adds:

- Redis configuration in `config.js`.
- `start:prod` now migrates and seeds database before running the server.
- Delete apk cache in Dockerfile after installing packages.

Closes https://github.com/debtcollective/parent/issues/211

### CODE REVIEW

File: views/includes/header.pug [hunk 1]
```diff
@@ -20,7 +19,8 @@ script.
       { text: 'Dispute Your Debt', href: '/dispute-tools' },
       { text: 'Campaigns', href: '#campaigns' },
       { text: 'Events', href: '#events' },
-      { text: 'My Disputes', href: '/disputes/my', roles: [!{JSON.stringify(basicRole)}] }
+      { text: 'My Disputes', href: '/disputes/my', roles: [!{JSON.stringify(basicRole)}] },
+      { text: 'Admin', href: '/admin/disputes', roles: [!{JSON.stringify(adminRole)}] }
```
Comment (marcondag) [inline views/includes/header.pug]: Just a heads up that this makes the header wrapping issue much worse

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "report": "yarn istanbul-combine -d coverage -p summary -r lcov coverage/{unit,integration,form-validations}/coverage.raw.json",
     "start": "node bin/server.js",
     "start:dev": "nodemon --inspect=9220 bin/server.js -i src/**/*",
-    "start:prod": "yarn db:migrate && node bin/server.js",
+    "start:prod": "yarn db:migrate && yarn db:seed && node bin/server.js",
```
Comment (marcondag) [inline package.json]: It's safe to seed every time we start?

File: Dockerfile [hunk 3]
```diff
@@ -2,9 +2,8 @@ FROM node:8.11.4-alpine
 
 WORKDIR /usr/src/app
 
-RUN apk update \
-    && apk upgrade \
-    && apk add build-base python git pdftk
+RUN apk add build-base python git pdftk --update \
```
Comment (marcondag) [inline Dockerfile]: Is it possible for us to just use [`--no-cache`](https://github.com/fluent/fluentd-docker-image/commit/bdbea02b09d13030937a3f31864f42d939eba912) here?

File: package.json (hunk 2 above)
Comment (orlando) [reply package.json]: Yeah, I ran the command and doesn't insert duplicated records

File: views/includes/header.pug (hunk 1 above)
Comment (orlando) [reply views/includes/header.pug]: Yeah, but admins asked to have it there. I missed this in the code review

File: Dockerfile (hunk 3 above)
Comment (orlando) [reply Dockerfile]: lol, was using https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache as a reference and didn't read till the end where they use `--no-cache`.

**Pull request merged**